tests/vegalite/v5/test_api.py

```diff
@@ -1065,3 +1072,35 @@ def test_validate_dataset():
     jsn = chart.to_json()
 
     assert jsn
+
+
+def test_polars_with_pandas_nor_pyarrow(monkeypatch: pytest.MonkeyPatch):
+    monkeypatch.delitem(sys.modules, "pandas")
+    monkeypatch.delitem(sys.modules, "numpy")
+    monkeypatch.delitem(sys.modules, "pyarrow", raising=False)
+
+    df = pl.DataFrame({"a": [1, 2, 3], "b": [4, 5, 6]})
+    _ = alt.Chart(df).mark_line().encode(x="a", y="b").to_json()
+    # Check pandas and PyArrow weren't imported anywhere along the way,
+    # confirming that the plot above would work without pandas no PyArrow
+    # installed.
+    assert "pandas" not in sys.modules
+    assert "pyarrow" not in sys.modules
+    assert "numpy" not in sys.modules
+
+
+@pytest.mark.skipif(
+    int(pd.__version__.split(".")[0]) < 2,
```

Comment:
```suggestion
    int(version("pandas")[0]) < 2,
```

Might be a little bit more readable, would require `from importlib.metadata import version` as well

Reply:
thanks for your review! I think that would still return a string though? so `version('pandas')[0]` would be `'1'` when pandas hits version 10 (granted, that's a while away 😄 )

Reply:
@MarcoGorelli you're absolutely right there

I suppose even with `version` you'd still need this for v10

```py
int(version('pandas').split(".")[0]) < 2
```

Reply:
sure, done, it is more readable 👍